src/App.tsx

```diff
@@ -352,6 +375,22 @@ const App: FC = () => {
           path="/ui/warnings"
           element={<ProtectedRoute outlet={<WarningList />} />}
         />
+        <Route
+          path="/ui/permissions/identities"
+          element={<ProtectedRoute outlet={<PermissionIdentities />} />}
+        />
+        <Route
+          path="/ui/permissions/identity/:authMethod/:name"
+          element={<ProtectedRoute outlet={<PermissionIdentityDetail />} />}
+        />
+        <Route
+          path="/ui/permissions/identity/:authMethod/:name/:activeTab"
+          element={<ProtectedRoute outlet={<PermissionIdentityDetail />} />}
+        />
+        <Route
+          path="/ui/permissions/lxd-groups"
```

Comment:
Later we will have Identity provider groups as a tab, better to keep the lxd prefix I think. Probably better to add the prefix to the tab.

Reply:
I see we want to distinguish groups and idp-groups. Though, I am not sure if the lxd prefix is supporting the disambiguity.

Reply:
My thinking is that `lxd-groups` indicates groups created in lxd where `idp-groups` means groups created within an idp platform. Terminology like auth-groups seems broad since it feels like it could be created in lxd or idp side. Maybe `lxd-auth-group`?